src/dbgshim/dbgshim.cpp

```diff
@@ -502,19 +508,19 @@ class RuntimeStartupHelper
         // Don't need to wake up and wait for the worker thread if called on it
         if (m_threadId != GetCurrentThreadId())
         {
-            // Wait for work thread to exit
-            WaitForSingleObject(m_threadHandle, INFINITE);
+            // Wait for work thread to exit for 60 seconds
+            WaitForSingleObject(m_threadHandle, 60 * 1000);
```

Comment:
How do we handle timeout here? Before we blocked, but now we have a failure mode and we don't ever communicate back (we have no output).

Reply:
This is called during UnregisterForRuntimeStartup() which I don't think needs a to return this timeout failure.  What is the debugger going to do if unregister fails?

Reply:
On second thought, yeah - the prior behavior just feels wrong.